server.js

```diff
@@ -1,9 +1,8 @@
 var express = require('express');
 var request = require('request');
 var cheerio = require('cheerio');
-var parseString = require('xml2js').parseString;
 var app = express();
-var env = process.env.NODE_ENV || 'development';
+var phantom = require('phantom');
```

Comment:
@davidlimcheng can you make sure to add this to your `package.json`? E.g. instead of using `npm install phantom`, do `npm install --save phantom`;